lib/st25r3911b/st25r3911b_common.h

```diff
@@ -48,12 +48,12 @@ extern "C" {
 /** FC in 4096/fc. */
 #define NFC_1FC_IN_4096FC 4096
 
-/** ST25R3911B NFCF LED pin. */
-#define ST25R3911B_NFCF_LED 4
-/** ST25R3911B NFCB LED pin. */
-#define ST25R3911B_NFCB_LED 28
-/** ST25R3911B NFCA LED pin. */
-#define ST25R3911B_NFCA_LED 29
+/** ST25R3911B NFCF LED ID. */
+#define ST25R3911B_NFCF_LED 1
```

Comment:
consider grouping these 3 definitions in enum type

Reply:
Right. Grouped in enum